src/components/carbon-provider/components.test-pw.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import type { ComponentMeta, StoryFn } from "@storybook/react";
```

Comment:
**suggestion**: my bad I must have missed this on my original review. Must have not even looked here, my apologies. Looks like there's still a lot of storybook-ish stuff in here. You can probably get rid of the Storybook imports. In fact most stuff. I'd just leave in one component which returns what you want, and leave in the theme const. Everything else imo, like the arg types can go.

Reply:
Awh good spot. I thought I'd deleted it all but I must have forgot to. Must have been another playwright refactor PR 😆

Reply:
No worries, again sorry to make you address it now I must have completely missed it first time around